traveller.yaml

```diff
@@ -1406,6 +1342,15 @@ definitions:
         items:
           $ref: '#/definitions/spatialConstraint'
 
+  productSetInformationUpdate:
+    description: Product Set Information Update
+    type: object
+    required:
+      - productSetId
+    properties:
+      status:
+        $ref: '#/definitions/productSetStatusUpdate'
+
```

Comment:
In the issueRequest we had the possibility to refer to an external transaction and a wallet transaction. Do we want to remove that?

Reply:
transactionReferences added in 7e5066a6fe6522033a305063d207beb74c9db959. Not sure about the walletTransactionId, that could potentially be scoped in another PID.

Reply:
I meant that in the issue operation we previously could add a transactionReference, in this case we need to send it in the productSetInformationUpdate since there is no wallet transaction.

Reply:
ok, so we want it in productSetInformationUpdate as well?

Reply:
Old issue also have walletTransactionId that we currently ignore but that we have told SIGMA to use, that is put the ID of the purse withdraw transaction here.

Are we happy with the following: To find the purchase transaction that is created in sales at issue, you have to search for a transaction with paymentReference matching the value that was used as transactionReference in the issue call? (there is currently no way do do this search using the sales API?). Or should the ID of the purchase transaction somehow be returned by issue?

Reply:
In traveller.yaml in Sales, we have the path '/productSet/{productSetId}/status' with GET and PUT. I can not find that path in the traveller.yaml from this branch.

The function to update status will be implemented by   '/productSet/{productSetId}' PATCH with productSetInformationUpdate.productSetStatusUpdate I suppose.

Was '/productSet/{productSetId}/status' removed in a previous update to the BoB API that we did not incorporate in sales or was it never part of the BoB API?

To get status, should GET '/productSet/{productSetId}' be used?

Reply:
> To get status, should GET '/productSet/{productSetId}' be used?

Yes.